data/json/items/melee.json

```diff
@@ -1834,6 +1834,22 @@
     },
     {
         "type":"GENERIC",
+        "id":"rock",
```

Comment:
Nope.  Two Items One id is not going to work.  Needs to be "rock_sharp" or something.  (That, and should probably change the name too, so people don't get confused.)